tests/phpunit/tests/theme.php

```diff
@@ -867,14 +867,14 @@ private function helper_requires_block_theme() {
 
 		// Skip if the block theme is not available.
 		if ( ! wp_get_theme( $block_theme )->exists() ) {
-			$this->markTestSkipped( "$block_theme must be available." );
+			$this->fail( "$block_theme must be available." );
```

Comment:
Same thing again. Failing tests should only be done when the code under test doesn't run correctly. Not when the environment in which the tests are run does not comply with the desired environment.